src/odemis/gui/cont/acquisition/fastem_acq.py

```diff
@@ -590,333 +983,349 @@ def on_acquisition_done(self, future):
             return
-        except Exception as exp:
+        except Exception:
             # leave the gauge, to give a hint on what went wrong.
             logging.exception("Acquisition failed")
-            self._reset_acquisition_gui("Acquisition failed (see log panel).", level=logging.WARNING)
-            return
+            self._reset_acquisition_gui(
+                "Acquisition failed (see log panel).", level=logging.WARNING
+            )
+        finally:
+            if self.chk_ebeam_off.IsChecked():
+                # Turn off e-beam if the checkbox is ticked
+                self._main_data_model.emState.value = STATE_OFF
 
 
 class FastEMCalibrationController:
     """
     Controls the calibration button to start the calibration and the process in the calibration panel
     in the FastEM overview and acquisition tab.
     """
-    def __init__(self, tab_data, calibration):
+
+    def __init__(
+        self,
+        tab_data,
+        main_tab_data,
+        tab_panel,
+    ):
         """
-        :param tab_data: (FastEMAcquisitionGUIData) The representation of the microscope GUI.
-        :param calibration: (FastEMCalibration) The object containing FastEM calibration related attributes.
+        :param tab_data: the FastEMSetupTab tab data.
+        :param main_tab_data: the FastEMMainTab tab data.
+        :param tab_panel: (wx.Frame) the frame which contains the viewport
         """
-        self._tab_data = tab_data
+        self._tab_data_model = tab_data
         self._main_data_model = tab_data.main
-        self.calibration = calibration
+        self._tab_panel = tab_panel
+        self._main_tab_data = main_tab_data
+        self._calib_gauge = self._tab_panel.gauge_calib
+        self._calib_label = self._tab_panel.lbl_calib
 
+        self.calibration_panel = SettingsPanel(
+            self._tab_panel.pnl_calib, size=(400, 80)
+        )
+        self._calib_1_lbl, self._calib_1 = self.calibration_panel.add_run_btn(
+            fastem.CALIBRATION_1
+        )
+        self._calib_1_lbl.SetToolTip(
+            "Optical path and pattern calibrations: Calibrating and "
+            "focusing the optical path and the multiprobe pattern. "
+            "Calibrating the scanning orientation and distance."
+        )
+        self._calib_1.SetName(fastem.CALIBRATION_1)
+        self._calib_2_lbl, self._calib_2 = self.calibration_panel.add_run_btn(
+            fastem.CALIBRATION_2
+        )
+        self._calib_2_lbl.SetToolTip(
+            "Dark offset and digital gain calibration (orange square): "
+            "Correcting between cell images for differences in "
+            "background noise and homogenizing across amplification differences."
+            " Place the region of calibration (ROC) orange square on empty part of "
+            " scintillator without a tissue."
+        )
+        self._calib_2.SetName(fastem.CALIBRATION_2)
+        self._calib_2_lbl.SetForegroundColour(FG_COLOUR_WARNING)
```

Comment:
The same colour is also applied to the ROC overlay, so it won't do any harm
https://github.com/nandishjpatel/odemis/blob/84cb7e8d44947ae85fedca80e34b4f8834865cfc/src/odemis/gui/model/main_gui_data.py#L689

Reply:
doesn't that just mean that if the FG_COLOUR_WARNING is changed it also changes in the ROC overlay. It will still show a different color than orange, which users will not expect